### PR TITLE
Fix/welcome screen

### PR DESCRIPTION
**Pull request by celinecamacho**

### CODE REVIEW

File: app/src/main/java/ch/epfl/balelecbud/RootActivity.java [hunk 1]
```diff
@@ -136,46 +147,46 @@ private void showScheduleFragment() {
 
     private void showPoiFragment() {
         Fragment fragmentPoi = PointOfInterestFragment.newInstance();
-        this.startTransactionFragment(fragmentPoi, "POI");
+        startTransactionFragment(fragmentPoi, PointOfInterestFragment.TAG);
```
Comment (Pialex99) [inline app/src/main/java/ch/epfl/balelecbud/RootActivity.java:150]: It is nicer now :-)

File: app/src/main/java/ch/epfl/balelecbud/view/welcome/WelcomeFragment.java [hunk 2]
```diff
@@ -40,14 +43,46 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+
+        SharedPreferences preferences = BalelecbudApplication
+                .getAppContext().getSharedPreferences(TAG, Context.MODE_PRIVATE);
+
+        if(isPageNumberSaved(preferences)) {
+            int pageNumber = preferences.getInt(TAG, -1);
+            if(pageNumber == -1) {
+                Log.d(TAG, "onStart: Failed to restore page");
+            }
+            else {
```
Comment (Pialex99) [inline app/src/main/java/ch/epfl/balelecbud/view/welcome/WelcomeFragment.java:64]:
```suggestion
            } else {
```

File: app/src/main/java/ch/epfl/balelecbud/view/welcome/WelcomeFragment.java [hunk 3]
```diff
@@ -40,14 +43,46 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+                welcomeViewPager.setCurrentItem(pageNumber);
+                //pageIndicator.getTabAt(pageNumber);
```
Comment (Pialex99) [inline app/src/main/java/ch/epfl/balelecbud/view/welcome/WelcomeFragment.java:66]: If you don't need it, remove it

File: app/src/main/java/ch/epfl/balelecbud/RootActivity.java (hunk 1 above)
Comment (karimassi) [reply app/src/main/java/ch/epfl/balelecbud/RootActivity.java:150]: I agree ! :-D

**Pull request merged**